src/Sql/Sql/help/Get-AzSqlDatabaseLongTermRetentionBackup.md

```diff
@@ -302,7 +302,7 @@ Parameter Sets: Location, ServerName, BackupName
 Aliases:
 
 Required: False
-Position: 0
+Position: Named
```

Comment:
So, what is the genesis of these changes?  Si this a lace where this change has been in for some time and we just never regenerated the help?  This is a breaking change, and so are many of the others- this highliughts the need to actually use the generation tools.

Reply:
Note: this is just a question, the associated changes are fine, provided this is just a case fo the help catching up with changes that are long released.

Reply:
Yeah, not my change and likely the person that changed it forgot to update help.  I will follow up on that to make sure.

Reply:
Confirmed with original committer that this change was intentional and he merely forgot to update help.